packages/e2e/test/frame_timing_summarizer_test.dart

```diff
@@ -16,7 +16,12 @@ void main() {
     rasterTimes = rasterTimes.reversed.toList();
     List<FrameTiming> inputData = <FrameTiming>[
       for (int i = 0; i < 100; i += 1)
-        FrameTiming(<int>[0, buildTimes[i], 500, rasterTimes[i]]),
+        FrameTiming(
+          buildStart: 0,
```

Comment:
Thanks for the updates! 
The new constructor introduced `vsyncStart`. Please use the following (or remove that `50+` and change the following `expect` accordingly: 
```Dart
        FrameTiming(
          vsyncStart: 0,
          buildStart: 50,
          buildFinish: 50+buildTimes[i],
          rasterStart: 500,
          rasterFinish: rasterTimes[i],
        ),
```

Reply:
I've added vsyncStart: 0 and buildStart: 0. For now the object that this code is testing doesn't seem to use the vsyncStart. I've left a TODO in the code to revisit this.

Reply:
Sounds good. I can handle this TODO.